docs/reference/fields.txt

```diff
@@ -897,6 +897,17 @@ can use in our model class as follows:
     field :location, type: Point
   end
 
+You may optionally declare a symbol mapping for the new field type in an initializer:
```

Comment:
This permits referencing the alias as a string also, not just as a symbol.

Reply:
To avoid confusion, I have removed word "symbol" and just said "You may optionally declare a mapping..."